README.md

```diff
@@ -1,25 +1,29 @@
 # Intro to Jasmine and JavaScript
+
+## Objectives
++ Run Jasmine Tests
++ Read Jasmine Tests
++ Use Debugger to figure out bugs in code
+
 ## Jasmine
 
 Jasmine is Flatiron's (and many other developers) testing library of choice for JavaScript.  It is modeled after RSpec, the de facto ruby testing library.  Jasmine offers a robust domain specific language that is easy to use and follow.  It's main use is for unit testing. In practice this means testing functions and objects in a program in isolation to assure they do what they're expected to.
 
-In Jasmine, to make set of tests for a particular subject you set it apart by creating a `describe` function with the first argument as the subject, and the second and `anonymous function`.  In the `block` of that `anonymous function` you can either go more specific with nested `describe` `block`s or an `it` function. `It` the actual test; it takes a string as the first argument about what it is testing and an `anonymous function` that contains an `expect` function that compares your code to the expected outcome.  
+## Instructions
+
+This lab is designed to get you comfortable with the syntax of Jasmine as well as JavaScript.All you need to do to complete it is make all the tests pass in the `spec/intro-spec.js`.  You'll be coding your solution in `code.js`.
```

Comment:
Is it worthwhile to break down one of the specs and explain what's happening? We haven't reviewed passing functions as parameters so the spec could look weird, but maybe point out which part is setting up the expectations?